plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/component/ComponentDetails.js

```diff
@@ -37,15 +42,29 @@ Ext.define('NX.coreui.view.search.SearchResultDetails', {
   /**
    * @override
    */
-  initComponent: function () {
+  initComponent: function() {
     var me = this;
 
     me.items = [
-      { xtype: 'nx-info', itemId: 'info1' },
-      { xtype: 'nx-info', itemId: 'info2' },
-      { xtype: 'nx-info', itemId: 'info3' }
+      {xtype: 'nx-info', itemId: 'info1'},
```

Comment:
as mentioned earlier, would prefer these itemIds to be more descriptive of contents if possible